config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -216,3 +216,42 @@ presubmits:
           env:
             - name: GOPATH
               value: /go
+  - name: pull-kubernetes-node-crio-1.18-e2e
```

Comment:
```suggestion
  - name: pull-kubernetes-node-crio-e2e
```

I would stick to the latest version

Reply:
Yes that makes sense. But [just like containerd](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/containerd.yaml#L68) we want to run the tests against multiple versions of CRIO. 

Since we are adding the CRIO tests for the first time, I thought it would greatly simplify the installation process if we could stay with the latest version that can be installed using simple `dnf` command on `fedora 31`